src/main/java/mate/academy/bookstore/repository/ShoppingCartRepository.java

```diff
@@ -0,0 +1,9 @@
+package mate.academy.bookstore.repository;
+
+import java.util.Optional;
+import mate.academy.bookstore.model.ShoppingCart;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface ShoppingCartRepository extends JpaRepository<ShoppingCart, Long> {
```

Comment:
```suggestion
@Repository
public interface ShoppingCartRepository extends JpaRepository<ShoppingCart, Long> {
```

I would suggest as a good practice to annotate with `@Repository` class you are implementing, because you already annotated `CartItemRepository`